packages/hardcaml-bloop/hardcaml-bloop.0.1.0/opam

```diff
@@ -17,7 +17,7 @@ depends: [
   "sattools"
   "bdd"
   "uchar"
-  "uutf"
+  "uutf" {< "1.0.0"}
```

Comment:
Should the constraint be reversed here?  The [opam-builder logs](http://opam.ocamlpro.com/builder/html/hardcaml-bloop/hardcaml-bloop.0.1.0/dca756685ef5ab57177792dd943dfa96) show that the failure occurred with `uutf.0.9.4` and the Travis build is also [failing with `uutf.0.93`](https://travis-ci.org/ocaml/opam-repository/jobs/205394925).